src/compass/utils/runconfig.py

```diff
@@ -78,7 +75,7 @@ def validate_group_dict(group_cfg: dict) -> None:
 
 
 def load_bursts(cfg: SimpleNamespace) -> list[Sentinel1BurstSlc]:
```

Comment:
I would suggest renaming this function, e.g., `runconfig_to_bursts()`, to avoid the confusion with `s1reader.load_bursts()`.